runtime/doc/treesitter.txt

```diff
@@ -523,6 +523,12 @@ associated with patterns:
     node's entire text should be re-parsed, including the text of its child
     nodes. By default, child nodes' text will be excluded from the injected
     document.
+    • `injection.self` - indicates that the node's text should be parsed with
+      the same language as the node itself.
+    • `injection.parent` - indicates that the captured node’s text should
+      be parsed with the same language as the node's parent LanguageTree.
+      This is only meant for injections that need to refer back to the
+      parent language to parse the node's text inside the injected language.
```

Comment:
```suggestion
```

Kinda redundant.

Reply:
Should I just say `This is only meant for injections.`? It might not be clear